types/error/SemanticErrorError.raml

```diff
@@ -0,0 +1,5 @@
+#%RAML 1.0 DataType
+(package): Error
+type: ErrorObject
+displayName: SemanticErrorError
```

Comment:
for the convention that we have in this repository I had to write "...ErrorError", not sure if it should be useful like this

Reply:
looks ugly but it's still correct